onnxruntime/test/util/test_utils.cc

```diff
@@ -70,14 +70,12 @@ void RunAndVerifyOutputsWithEP(const ORTCHAR_T* model_path, const char* log_id,
                                std::unique_ptr<IExecutionProvider> execution_provider,
                                const NameMLValMap& feeds) {
   // read raw data from model provided by the model_path
-  ONNX_NAMESPACE::ModelProto model_proto;
-  ASSERT_STATUS_OK(onnxruntime::Model::Load(model_path, model_proto));
-  std::string model_data;
-  model_proto.SerializeToString(&model_data);
+  std::ifstream stream(model_path, std::ios::in | std::ios::binary);
+  std::vector<uint8_t> model_data((std::istreambuf_iterator<char>(stream)), std::istreambuf_iterator<char>());
```

Comment:
If you want you can simply use 
```
std::string model_data( ( std::istreambuf_iterator<char>( stream ) ), std::istreambuf_iterator<char>() );
```
such that the RunAndVerifyOutputsWithEPViaModelData can use `const std::string&`